template/default.tmpl

```diff
@@ -46,9 +46,19 @@ Alerts Resolved:
 {{ define "opsgenie.default.source" }}{{ template "__alertmanagerURL" . }}{{ end }}
 
 
-{{ define "victorops.default.state_message" }}{{ template "__subject" . }} | {{ template "__alertmanagerURL" . }}{{ end }}
-{{ define "victorops.default.monitoring_tool" }}{{ template "__alertmanager" . }}{{ end }}
+{{ define "victorops.default.state_message" }}{{ template "__alertmanager" . }}: {{ template "__alertmanagerURL" . }}
```

Comment:
I'm using `__subject` as `entity_display_name` now, and using a more verbose `state_message`(similar to `opsgenie.default.description`) instead of `__subject`.
In other words having `__subject` in 2 different fields is not very useful.